data_collection/gazette/spiders/rj_sao_joao_de_meriti.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+import json
+import re
+
+import scrapy
+from dateutil.rrule import MONTHLY, rrule
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpSaoJoaoDeMeritiSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3305109"
+    name = "rj_sao_joao_de_meriti"
+    allowed_domains = ["meriti.rj.gov.br"]
+
+    URL_BEFORE_2018 = "https://meriti.rj.gov.br/dom_mp/"
+    URL_FROM_2018 = (
+        "https://transparencia.meriti.rj.gov.br/diario_oficial_get.php?mesano="
+    )
+
+    start_date = datetime.date(2011, 9, 30)
+    end_date = datetime.date.today()
+
+    def start_requests(self):
+        janeiro_2018 = datetime.date(2018, 1, 1)

```

Comment:
**nitpick**: Our code should be written preferably in English, so `january_2018` is a better name for this variable.